flexmeasures/data/models/generic_assets.py

```diff
@@ -101,7 +101,7 @@ def __acl__(self):
         """
         return {
             "create-children": f"account:{self.account_id}",
-            "read": f"account:{self.account_id}"
+            "read": self.owner.__acl__()["read"]
```

Comment:
Still needs to update the docstring to match. Line 2 could become:

"For non-public assets, we allow reading to whoever can read the account, and editing for every user in that account."